src/app/products/products.component.html

```diff
@@ -1,17 +1,31 @@
 <h2>Products</h2>
 <app-product-detail [product]="product"></app-product-detail>
 <div class="productsContainer">
-  <div class="productCard" *ngFor="let product of products" (click)="onSelect(product)">
+  <div class="productCard" *ngFor="let product of products; let i = index" (click)="onSelect(product)">
     <img src="{{ product.image }}">
     <h3>{{ product.name }}</h3>
-    <div>{{ product.size }}</div>
-    <div>{{ product.price }} zl</div>
+    <hr>
+    <div>
+      <span class="productCardPropHeader">Color: </span>
+      <span class="productCardPropValue">{{ product.size }}</span>
+    </div>
+    <div>
+      <span class="productCardPropHeader">Price </span>
+      <span class="productCardPropValue">{{ product.price }}</span> zl
+    </div>
+    <div>
+      <span class="productCardPropHeader">Quantity: </span>
```

Comment:
We will need to fix how this works, as this should be stock amount ideally. I think we can remove it for the time being as it is slightly confusing. As they all have starting quantity -> 1.